client_src/js/services/local-storage.service.js

```diff
@@ -0,0 +1,15 @@
+import { ApplicationError } from "../exceptions/application-error.handler";
+import { ClientErrors } from "../exceptions/octofarm-client.exceptions";
+
+export function removeLocalStorage(key) {
+  if (!!key) throw new ApplicationError(ClientErrors.FAILED_VALIDATION_KEY);
```

Comment:
This is not validation for me. This is plain checking. The error does not make it more obvious what the problem is, so if we had to track bugs afterwards we'd be having more trouble than a `throw new Error('localStorage key '${key}' was not provided');`

Reply:
Yeah I know, there's no validation written yet and it's for another PR.